machine.go

```diff
@@ -119,6 +119,8 @@ func NewMachine(mac net.HardwareAddr, broker *Broker) *Machine {
 			{Name: "served_ipxe_over_tftp", Src: []string{"point_pxe_to_ipxe_over_tftp"}, Dst: "served_ipxe_over_tftp"},
 			{Name: "point_ipxe_to_http_boot", Src: []string{"served_ipxe_over_tftp"}, Dst: "point_ipxe_to_http_boot"},
 
+			{Name: "http_fetch_uki", Src: []string{"http_boot", "point_ipxe_to_http_boot"}, Dst: "http_fetch_uki"},
+
 			{Name: "os_init", Src: []string{"http_boot", "point_ipxe_to_http_boot"}, Dst: "os_init"},
```

Comment:
```suggestion
			{Name: "os_init", Src: []string{"http_fetch_uki"}, Dst: "os_init"},
```

I think we now expect all transition to os_init go via http_fetch_uki.

Reply:
(...and not tidying up the `Src` part of the transition graph causes reset() to be triggered more often, which is undesirable and implies the system did not follow an expected boot flow.)